packages/coze-taro/test/stubs.ts

```
@@ -1,6 +1,37 @@
 /* eslint-disable @typescript-eslint/no-magic-numbers -- ignore */
 /* eslint-disable security/detect-object-injection -- ignore */
-import { Events } from '@tarojs/taro';
+
+export class Events {
+  private map = new Map<string, Array<Function>>();
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)